spec/cocoapods/generate/configuration_spec.rb

```diff
@@ -7,6 +7,9 @@
   it { should_not be_nil }
 
   describe_method :to_h do
+    # ensure we do not recusrively catch integration test podspecs.
+    let(:kwargs) { { podspec_paths: [Pathname('./spec/cocoapods/generate')] } }
```

Comment:
because we now recursively find podspecs, I Had to update this test to ensure it only operates from within the folder it cares for.